pages/index.tsx

```diff
@@ -56,6 +57,7 @@ Home.getLayout = function getLayout(page) {
         {page}
       </div>
       <Footer />
+      <ReactTooltip type="dark" effect="solid" place="left" backgroundColor='#000000'/>
     </Layout>
   );
```

Comment:
Also, because the tooltip is in `Home.getLayout`, it will only appear on the Home page. So if someone tries to submit a track from any other page, they won't see the tooltip. 

I think your intention was for the tooltip to be rendered on every page. If you want the same component to be on every page, see where I put `ToastContainer` in `pages/_app.tsx`.

But as I wrote above, this is unnecessary for this tooltip.